prime-router/settings/organizations.yml

```diff
@@ -225,7 +226,7 @@
         operation: MERGE
         numberPerDay: 1440 # Every minute
         initialTime: 00:00
-      translation: !<HL7>
+      translation:
```

Comment:
removed these YAML tags as they were not considered valid yaml in intellij (although they are...) and the values appear duplicated within the `translation.type` property.

Reply:
These tags are no longer emitted when getting yaml settings. They were an artifact of how the jackson yaml mapper is set up.